### DISCUSSION TITLE
Is is possible to set the selectionStart when using setFocus?

### DISCUSSION THREAD

**Discussion by r1chm8** [react-hook-form/react-hook-form#9458] [Q&A]
I see it's possible to pass options to `setFocus`  I'm wondering if there's anyway to set the `selectionStart` to the end for example:

```
setFocus('my-textarea', {
  selectionStart: 'end',
});
```

If not, is there a nice way to achieve the same result?

Thanks

**Comment by bluebill1049** [comment]
you can submit a feature request for this.
